### DISCUSSION TITLE
Feature Request: Support for Tab Groups

### DISCUSSION THREAD

**Discussion by bpan2020** [doublecmd/doublecmd#2019] [Suggestions]
Since Double Commander is designed as a twin-panel file manager, I propose extending the 'panel' concept to support multiple groups of tabs. Each group would maintain its own panel layout, allowing users to switch between different groups as needed. 

Just like this.

![图片](https://github.com/user-attachments/assets/45cb591f-2e8f-4579-ac4c-939d4480a71b)

![图片](https://github.com/user-attachments/assets/044a95fc-eba7-45c2-a11a-de2eb5af4236)

![图片](https://github.com/user-attachments/assets/298f4bd5-1e85-4417-9d0a-08ba95fdb71f)

**Comment by j2969719** [comment]
https://doublecmd.github.io/doc/en/configuration.html#ConfigFavoriteTabs

**Comment by bpan2020** [comment]
Got it, thanks.
